deploy/mod/deploy.py

```diff
@@ -156,7 +156,8 @@ def install_binary(self, version=""):
                        'firefly': '0.80',
                        'giant': '0.87',
                        'hammer': '0.94',
-                       'infernalis': '9.2'}
+                       'infernalis': '9.2',
+		       'jewel': '10.2'}
```

Comment:
Also need to updated at tuner.py apply_version function, and maybe you can test on webui / by python tuner.py to verify? Thanks